src/main/scala/io/iohk/ethereum/jsonrpc/JsonMethodsImplicits.scala

```diff
@@ -94,15 +94,15 @@ trait JsonMethodsImplicits {
 
     for {
       from <- input.get("from") match {
-        case Some(JString(s)) => extractAddress(s)
+        case Some(JString(s)) if s.nonEmpty => extractAddress(s)
+        case Some(JString(_)) => extractAddress("0x0")
```

Comment:
that line is invalid I think, no one should be able to send tx from "0x0" address